tests/features/warmshowers/create_account.feature

```diff
@@ -0,0 +1,55 @@
+#language:en
+
+Feature: I can create an account
+	In order to become a WarmShowers member
+	As a new user
+	I can create an account
+
+Scenario Outline: As a new user, I can view the account creation page in my language

```

Comment:
I'm sorry I confused you a bit by asking for more descriptive Scenario names, they don't have to be in Gherkin style, in fact I think they're much more readable in good old English, so I'd make this one:
`Viewing the account creation page`